go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ElrondNetwork/elrond-go
 go 1.13
 
 require (
-	github.com/ElrondNetwork/arwen-wasm-vm v1.2.14
+	github.com/ElrondNetwork/arwen-wasm-vm v1.2.15-0.20210531084552-d81a17a3049d
```

Comment:
need a proper release. It will be funny that we will always need to change the implementation for v1.2 - even when it is about some interface changes.